### ISSUE TITLE
Incorrect usage of CreateObject for component usage

### ISSUE THREAD

**Issue by TwitchBronBron** [rokucommunity/bslint#61]
It appears that the `LINT4002` bslint rule incorrectly scans for the component name _anywhere_ in the `CreateObject` call. Instead, it should only look for them directly following a "roSGNode". 

```xml
<?xml version="1.0" encoding="utf-8" ?>
<component name="Parent" extends="Group">
</component>
```

```brightscript
? createObject("Parent") 'bslint should NOT consider this valid usage of the `Parent` component
? createObject("roSGNode", "Parent") ' THIS is the proper usage of `Parent` component 
```
